scss/styles.scss

```diff
@@ -1,3 +1,35 @@
-.title {
-  color: red;
+@import "colors";
+
+* {
+  font-family: "Roboto", sans-serif;
+}
+
+%gray-text {
+  color: $gray;
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: bold;
+  margin-bottom: 26px;
+
+  .book-genre {
+    @extend %gray-text;
+
+    display: inline;
+    font-size: 24px;
+    font-weight: bold;
+  }
+}
+
+.book-attribute {
+  @extend %gray-text;
+
+  display: block;
+  font-size: 20px;
+  margin-top: 38px;
```

Comment:
try to always use margin to the same orientation. On the title you use "margin-bottom",  stick with that. It can lead to problems because the two margins will stack